devices/Chs6540/Chs6540.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.IO;
+
+namespace Iot.Device.Chs6540
+{
+    /// <summary>
+    /// Chs6540 touch screen
+    /// </summary>
+    public class Chs6540 : IDisposable
+    {
+        private I2cDevice _i2cDevice;
+        private byte[] _data = new byte[2];
+        private byte[] _toReadPoint = new byte[6];
+
+        /// <summary>
+        /// Chs6540 I2C Address
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x2E;
+
+        /// <summary>
+        /// Creates a new instance of the Chs6540
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device used for communication.</param>
+        public Chs6540(I2cDevice i2cDevice)
+        {
+            _i2cDevice = i2cDevice ?? throw new ArgumentException(nameof(i2cDevice));        
```

Comment:
No need to "waste" flash with the arg name. Please use the empty constructor and add a comment entry for the ArgumentException with the explanation.